.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,28 @@
+#### Problem Description
+[Describe the issue and explain **why** it is a problem or what the expected output should be]
+
+#### Minimal Reproducible Example
+[Make it simple, copy-pastable and include any library calls]
+
+```R
+
+```
+
+[Include the Image(s) if any, of the Minimal Reproducible Example]
```

Comment:
I think you can delete both of these statement in favour of recommending the reprex package